docs/advanced-options.md

```diff
@@ -0,0 +1,134 @@
+# Advanced Options
+
+## Contents
+[Video Options](#video-options)  
+[Dynamic Discovery](#dynamic-discovery)  
+[Camera Paths](#keep-the-paths-of-existing-camera-up-to-date) 
+[RTSP Server](#configurable-rtsp-server-hostname-and-port)   
+
+## Video options
```

Comment:
This stuff in RtspStreamingGuide as well, so maybe delete here?

Reply:
That brings up a question I had about this @mpvats-intel 
Do we want these walkthrough guides to be independent of the other docs? Because as the ONVIF docs sit right now (and this could be easily changed), the main readme is a landing page, and you can choose to go to a full start to finish walkthrough, but also this content is covered in individual files so they can be easily referenced independently and easily. 
This is how it looks in ONVIF
# Getting Started

Learn how to configure and run the service by following these [instructions](./doc/setup.md). 

For a full walkthrough of using the default images, use this [guide.](./doc/guides/SimpleStartupGuide.md)  

For a full walktrhough of building custom images, use this [guide.](./doc/guides/CustomStartupGuide.md)

Reply:
what does @ajcasagrande think?

Reply:
@ChristianDarr-personal I see what you're saying and can go either way.